src/frameworks/index.ts

```diff
@@ -222,6 +222,8 @@ function scanDependencyTree(searchingFor: string, dependencies = {}): any {
  */
 export function findDependency(name: string, options: Partial<FindDepOptions> = {}) {
   const { cwd, depth, omitDev } = { ...DEFAULT_FIND_DEP_OPTIONS, ...options };
+  const env: any = Object.assign({}, process.env);
```

Comment:
Super nit, but do we even need `Object.assign` in the world of splats? I've just started using `{...original}` for shallow copies and `utils.cloneDeep()` for deep copies.

Reply:
I was trying to override NODE_ENV a few ways, this is just where I ended up and left it.